packages/ui/.storybook/main.ts

```diff
@@ -8,7 +8,6 @@ const config: StorybookConfig = {
     '@storybook/addon-links',
     '@storybook/addon-essentials',
     '@storybook/addon-interactions',
-    '@storybook/addon-styling',
```

Comment:
In recent versions of Storybook this plugin is no longer needed. Removed due to it causing it error related to PostCSS. There doesn't seem to be a negative effect due to removing it.